test/tests/neml2/tests

```diff
@@ -4,11 +4,12 @@
   [neml2_error]
     type = RunException
     input = 'fem.i'
-    cli_args = 'neml2_input=error'
+    cli_args = 'neml2_input=error Executioner/num_steps=1'
```

Comment:
I don't have an objection to this, but I'm not sure why this would help. I assume it hits the error and exits out in the first step anyway?

Reply:
Yep, that was the only reason -- I don't want to run the heavy valgrind test for an unnecessarily long time.

Reply:
Oh I misunderstood. I think I tried `Executioner/abort_on_solve_fail=true`, but then the RunException test fails with "no crash", which I didn't quite understand. So I went back and changed to this `num_steps=1` but now that I think about it, this probably does nothing, because the simulation will keep running with reduced time steps, until it hits dt_min.

Reply:
I don't know if there is a "fail on exception" sort of option. If so I'd like to use that. But for now this probably isn't very important.